core/primitives/src/transaction.rs

```diff
@@ -211,6 +211,27 @@ impl BorshDeserialize for Transaction {
     }
 }
 
+#[derive(Debug)]
+pub struct ValidatedTransaction(SignedTransaction);
```

Comment:
This badly needs a doc-comment to list out what exactly `Validated` here entails and more importantly what it does *not* entail.

Reply:
Addressed.